rules/windows/privilege_escalation_windows_service_via_unusual_client.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/02/07"
+maturity = "production"
+updated_date = "2022/02/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a Windows service by an unusual client process. Services may be created with administrator
+privileges but are executed under SYSTEM privileges, so an adversary may also use a service to escalate privileges from
+administrator to SYSTEM.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Windows Service Installed via an Unusual Client"
+note = """## Config
+
+`System Security Extension` audit subcategory need to be enabled to log event 4697 `A service was installed in the system`.
+
+"""
+references = [
+    "https://www.x86matthew.com/view_post?id=create_svc_rpc",
+    "https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4697",
+]
+risk_score = 73
+rule_id = "55c2bf58-2a39-4c58-a384-c8b1978153c2"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.code:"4697" and (winlog.event_data.ClientProcessId:"0" or winlog.event_data.ParentProcessId:"0")
```

Comment:
Maybe add event.action to give a bit more context

Reply:
good suggestion (done).